extensions/wikia/CategoryPage3/styles/category-page3.scss

```diff
@@ -0,0 +1,195 @@
+@import 'extensions/wikia/DesignSystem/node_modules/design-system/dist/scss/wds-mixins/index';
+@import 'extensions/wikia/DesignSystem/node_modules/design-system/dist/scss/wds-variables/index';
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/columns';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+
+// It's either that or !important for multiple selectors
+.WikiaArticle {
+	.category-page {
+		$space-between-groups: 48px;
+
+		&__trending-pages-header {
+			font-size: $wds-font-size-base;
+			font-weight: $wds-font-weight-bold;
+			margin-bottom: 12px;
+			margin-top: 50px;
+			text-transform: uppercase;
+		}
+
+		&__trending-pages {
+			// Grid is totally broken in IE11, -ms-inline-flexbox is better
+			display: -ms-inline-flexbox;
+			-ms-flex-wrap: wrap;
+
+			border-bottom: 1px solid $color-page-border;
+			display: grid;
+			grid-gap: 20px;
```

Comment:
Does `grid-gap` work OK in Safari: https://developer.mozilla.org/en-US/docs/Web/CSS/gap#Browser_compatibility ?